x-pack/legacy/plugins/ml/server/models/results_service/results_service.js

```diff
@@ -200,6 +200,75 @@ export function resultsServiceProvider(callWithRequest) {
 
   }
 
+  // Obtains the latest bucket result timestamp by job ID.
+  // Returns data over all jobs unless an optional list of job IDs of interest is supplied.
+  // Returned response consists of latest bucket timestamps (ms since Jan 1 1970) against job ID
+  async function getLatestBucketTimestampByJob(jobIds) {
```

Comment:
for readability, giving `jobIds` a default value of `[]` would show that it's an optional parameter.
the `if` on line 215 would then not need the first falsy check.